nav2_costmap_2d/include/nav2_costmap_2d/costmap_2d_publisher.h

```diff
@@ -35,16 +35,16 @@
  * Author: Eitan Marder-Eppstein
  *         David V. Lu!!
  *********************************************************************/
-#ifndef COSTMAP_2D_COSTMAP_2D_PUBLISHER_H_
-#define COSTMAP_2D_COSTMAP_2D_PUBLISHER_H_
+#ifndef nav2_costmap_2d_nav2_costmap_2d_PUBLISHER_H_
```

Comment:
We can have a separate PR to clean up header guards and address any linting issues.